apps/aehttp/priv/swagger.yaml

```diff
@@ -3413,6 +3409,23 @@ definitions:
       - fee
       - auth_data
       - tx
+  OldDelegates:
```

Comment:
Perhaps something like `DelegatesList` would be a better name, since it's also referenced by `NewDelegates`?